_docs/board_front.md

```diff
@@ -36,7 +36,14 @@ General Images: Every image in a board can be called by simply using its name en
           - gain-energy(X): Gain X Energy
           - make-fast: One of your Powers may be Fast
           - add-presence(X): Add a Presence up to X Range
-          - add-presence(X,Y): Add a Presence limited to Y Land type up to X Range
+		  - add-presence(X,Y): Add a Presence with Y conditions at X Range
+          - add-presence(X,Y,Z,...,and/or): Add a Presence with multiple Y, Z, ... conditions at X Range, the last parameter must be 'or' or 'and'.
+			- Y, Z, and so on can be terrain types (including dual types), tokens, invaders, invader pieces, dahan, blight, etc.
+			- If using more than just Y, the last parameter must be 'or' or 'and'. 
+			- For example, add-presence(3,jungle,beast,or) as seen on Sharp Fangs or add-presence(2,town,city,blight,or) from Vengeance
+		  - add-presence(2,text,your_text_here): adds arbitrary text in case the desired option is not implemented
+		  - add-presence(2,token,and/or,token_type): allows for adding tokens in addition to or instead of presence
+		    - For example, add-presence(3,token,and,beast) is Many Minds, add-presence(1,token,or,disease) is Vengeance
```

Comment:
The original file only uses spaces for indentation, but you use tabs. Most likely you don't see this in your local editor since your tab size is 4, but in the github interface with a tab size of 8 it breaks the indentation. Let's keep to one standard.